types/parse/index.d.ts

```diff
@@ -343,9 +344,9 @@ declare namespace Parse {
         save(attrs?: { [key: string]: any } | null, options?: Object.SaveOptions): Promise<this>;
         save(key: string, value: any, options?: Object.SaveOptions): Promise<this>;
         save(attrs: object, options?: Object.SaveOptions): Promise<this>;
-        set(key: string, value: any, options?: Object.SetOptions): boolean;
-        set(attrs: object, options?: Object.SetOptions): boolean;
-        setACL(acl: ACL, options?: SuccessFailureOptions): boolean;
+        set(key: string, value: any, options?: Object.SetOptions): this | boolean;
+        set(attrs: object, options?: Object.SetOptions): this | boolean;
+        setACL(acl: ACL, options?: SuccessFailureOptions): this | boolean;
```

Comment:
The documented return type appears to be `boolean`: https://parseplatform.org/Parse-SDK-JS/api/master/Parse.Object.html#setACL

Maybe add a test showing an example of a non-boolean return?  (Guess: these calls are chainable.)

Reply:
Oops, I missed your link to the actual source.  Looks like it's actually `this | false`.

Reply:
@amcasey You're right! I'm going to change all occurrences to `this | false`.